stubs/PyMySQL/pymysql/_auth.pyi

```diff
@@ -0,0 +1,21 @@
+from _typeshed import Incomplete
+from typing import Final
+
+DEBUG: Final[bool]
+SCRAMBLE_LENGTH: Final[int]
+sha1_new: Incomplete
+
+def scramble_native_password(password: str | bytes | None, message: str | bytes | None) -> bytes: ...
```

Comment:
Looks like any buffer works, not just bytes. Use `_typeshed.ReadableBuffer` instead of `bytes` for the parameter types.

```
In [16]: pymysql._auth.scramble_native_password(memoryview(b"x"), memoryview(b"x"))
Out[16]: b'\xe0\x1e\xa34\xcd"n]4\x06b\x06\xe3\xb1\x01iA7\xd5\r'
```